File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

const line1 = input[0].split(' ');
let N = line1[0];
let X = line1[1];

let arr = input[1].split(' ');
let ans = '';
for(let num of arr) {
    if(X > +num) ans += num + " ";
}

console.log(ans.toString());

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main{
    static BufferedReader br =
            new BufferedReader(new BufferedReader(new InputStreamReader(System.in)));
    public static void main(String[] args) throws IOException {
        String[] input = br.readLine().split(" ");
        int N = Integer.parseInt(input[0]);
        int X = Integer.parseInt(input[1]);

        input = br.readLine().split(" ");
        for(String s : input) {
            int num = Integer.parseInt(s);
            if(num < X) System.out.print(num+ " ");
        }

    }
}
